tools/include/eosio/gen.hpp

```diff
@@ -580,6 +588,10 @@ struct generation_utils {
       return builtins.count(_translate_type(t)) >= 1;
    }
 
+   inline bool is_reserved_type( const std::string& t ) {
```

Comment:
This doesn't have the same calling style as the rest of the `is_*_type` functions.  Is that by design?

Reply:
Not intentionally, you're right it should either take a QualType or have a different naming format.